pages/form/TravelMethod.jsx

```diff
@@ -78,8 +78,21 @@ export default function Question4() {
                 Select the ways you generally travel to work.
               </Text>
             </Box>
-            <TravelMethodButtons handleTransportMode={handleTransportMode} />
-
+            <TravelMethodButtons handleTransportMode={handleTransportMode}  />
+{/* Collapse on carpool selection*/}
+<Collapse in={transportMode === "Carpool"}>
```

Comment:
please fix indentation of code here:
```suggestion
        {/* Collapse on carpool selection*/}
        <Collapse in={transportMode === "Carpool"}>
```